guides/v2.2/frontend-dev-guide/themes/js-bundling.md

```diff
@@ -1 +1,135 @@
-../../../v2.1/frontend-dev-guide/themes/js-bundling.md
\ No newline at end of file
+---
+group: fedg
+subgroup: A_Themes
+title: JavaScript Bundling
+menu_title: JavaScript Bundling
+menu_order: 9000
+version: 2.2
+github_link: frontend-dev-guide/themes/js-bundling.md
+functional_areas:
+  - Frontend
+  - Theme
+---
+
+JavaScript bundling is an optimization technique you can use to reduce the number of server requests for JavaScript files.
+Bundling accomplishes this by merging multiple JavaScript files together into one file to reduce the number of page requests.
+
+## Enable JavaScript bundling
+
+Magento must be set to [production mode][production-mode] in order for bundling to work.
+
+To activate JavaScript bundling:
+
+- Enter the following command to make
+    ```bash
+    php -f bin/magento config:set dev/js/enable_js_bundling 1
+    ```
+    
+<div class="bs-callout bs-callout-info" markdown="1">
+Enable Javascript Bundling configuration is not longer available in Magento version 2.2 and next version. 
+</div>    
+
+To optimize bundling, set the following settings to `Yes`:
+
+* `Javascript Settings -> Merge JavaScript Files`
+* `Javascript Settings -> Minify Javascript Files`
+* `Static Files Settings -> Sign Static Files`
+
+## How bundling works in Magento
+
+By default, Magento bundles all JavaScript files and loads them synchronously on every page load.
```

Comment:
Isn't this only the case when bundling is enabled and not necessarily "by default"? By default individual modules are loaded asynchronously. I think it's **really** important to make it clear that bundling Javascript files moves most modules to be render-blocking in the head, though, so let's keep that part!

Something like:

> When bundling is enabled, Javascript files are loaded in synchronously in the head on every page load. They become render-blocking assets, unlike the default where every module is requested individually and asynchronously.

Reply:
Done.